Move must-gather folder to a predictable name

cescgina opened the pull request:
The must-gather output has a random sha suffix which makes it difficult
to link directly to folders inside of it in zuul artifacts. This commit
renames the must-gather output folder with a predictable
name, which then can be linked.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:

lewisdenny commented:
Hi @cescgina, would overriding the `cifmw_os_must_gather_output_dir`[1] default work? This would be a more clean approach if so. 

If not, could you please update the title and description of this PR as I see you have changed from symlink to just moving the directory and deleting the old one.

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/os_must_gather/README.md?plain=1#L12C1-L12C100

cescgina commented:
> Hi @cescgina, would overriding the `cifmw_os_must_gather_output_dir`[1] default work? This would be a more clean approach if so.
> 
> If not, could you please update the title and description of this PR as I see you have changed from symlink to just moving the directory and deleting the old one.
> 
> [1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/os_must_gather/README.md?plain=1#L12C1-L12C100

If I understood correctly the behaviour of must-gather, the `cifmw_os_must_gather_output_dir` variable just sets where the output is stored, but inside `cifmw_os_must_gather_output_dir` it still creates the `quay-io-openstack-k8s-operators-openstack-must-gather-sha256*` folder.

I modified  the commit message but I did forget to update the PR body and title, I'll do it in the next update, thanks!

marios reviewed (commented):
this looks good is it working as expected?

also please update the pull request title to align with the latest commit message  (you are no longer symlinking)

cescgina commented:
> this looks good is it working as expected?
> 
> also please update the pull request title to align with the latest commit message (you are no longer symlinking)

Not quite working yet, it's timing out while copying the folder, updated the PR to use `mv` instead and updated PR title and description

pablintino commented:
/lgtm

raukadah reviewed (commented):
/approve

raukadah commented:
recheck crc start issue
